ush/forecast_postdet.sh

```diff
@@ -2,6 +2,7 @@
 
 # Disable variable not used warnings
 # shellcheck disable=SC2034
+
```

Comment:
```suggestion
```
Adding this blank line disables the shellcheck warnings on the whole script.
Without this line, shellcheck is disabled only for this function.